src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerSearchPostTest.java

```diff
@@ -218,9 +212,7 @@ void casesSearchPostEndpointEventText() throws Exception {
     void casesSearchPostEndpointJudgeName() throws Exception {
 
         CourthouseEntity courthouseEntity = dartsDatabase.createCourthouseUnlessExists("SWANSEA");
-        UserAccountEntity testUser = dartsDatabase.getUserAccountStub()
-            .createAuthorisedIntegrationTestUser(courthouseEntity);
-        when(mockUserIdentity.getUserAccount()).thenReturn(testUser);
+        when(authorisationApi.getListOfCourthouseIdsUserHasAccessTo()).thenReturn(List.of(courthouseEntity.getId()));
```

Comment:
Is there a negative test here? ie. checking that the cases aren't returned for a courthouse the user does not have permission on?

Reply:
done